ci/cluster-api/vsphere/templates/cluster.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: cluster.x-k8s.io/v1alpha2
+apiVersion: cluster.x-k8s.io/v1alpha3
```

Comment:
I did not see you mentioned clusterapi yaml changes in change log?

Reply:
I didn't include it because the version of ClusterAPI we use for creating test clusters doesn't directly impact users. Actually, the only reason why it is part of the PR is because it is a requirement for some CI jobs to run.